src/components/mainContent/DirectionButtons.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import "./mainContent.css";
+import arrows from "../../assets/upArrow.svg";
+import content from "../../assets/labContent.js";
+export default function DirectionButtons(props) {
+  function handleBackward() {
+    let previousSection = content[props.visibleSections - 1];
+    let previousLength = undefined;
+    if (previousSection && previousSection.post)
+      previousLength = previousSection.post.bodyText.length;
+    else if (previousSection && previousSection.followups)
+      previousLength = previousSection.followups.length;
+
+    /*Should we use SWITCH statements instead */
```

Comment:
If there's only two statements, an `if` is probably fine - in fact, if it's a true `if-elseif-noaction` ladder, this is probably the better way to do it.